src/Services/ExpressCheckout.php

```diff
@@ -138,6 +138,27 @@ public function doExpressCheckoutPayment($data, $token, $payerid)
 
         return $response;
     }
+    
+    /**
+     * Function to perform DoCapture PayPal API operation.
+     *
+     * @param  string $authorization_id Transaction ID
+     * @param  float  $amount           Amount to capture
+     * @param  string $complete         Indicates whether or not this is the last capture.
+     * @param  array  $data             Optional request fields
+     * @return array
+     */
+    public function doCapture($authorization_id, $amount, $complete = 'Complete', $data = [])
+    {
+        $response = $this->doPayPalRequest('DoCapture', array_merge($data, [
+            'AUTHORIZATIONID' => $authorization_id,
+            'AMT'             => $amount,
+            'COMPLETETYPE'    => $complete,
+            'CURRENCYCODE'    => $this->currency,
+        ]));
+
```

Comment:
As per my last comment, this should be changed to the following then:

        $response = $this->doPayPalRequest('DoCapture', [
             'AUTHORIZATIONID' => $authorization_id,
             'AMT'             => $amount,
             'COMPLETETYPE'    => $complete,
             'CURRENCYCODE'    => $this->currency,
         ]);